menu.xml

```diff
@@ -25,7 +25,6 @@
 			</LED_Color>
 			<LED_Color text="Multicolor">
 				<Multicolor text="Add Color"></Multicolor>
-				<Multicolor text="Confirm"></Multicolor>
```

Comment:
"Confirm" was used to set colors to Multicolor. Naming might be unclear and inconsistent with the selection of the rest of the colors, but without it you can't change colors from screen.

Reply:
Ah.  It wasn't clear that it what it was for.  I'll put it back, in the dynamic menu portion.